lib/loader/index.js

```diff
@@ -14,9 +14,13 @@
  */
 
 var global = require('../global');
+var namespacer = require('../namespacer');
 var doc = global.document;
 
 var DEFAULT_TIMEOUT = 10000;
+var SHARED_NAMESPACE_NAME = 'BV_LOADER_TRACKING';
```

Comment:
Since this is open and not BV-specific, I think I'd rather see it default to not adding to a namespace. E.g. keep a local reference to a hash object in loadedUrls, and only add it to a namespace if provided with one.